utils/chem/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# Diag B
+ecbuild_add_executable( TARGET chem_diagb.x
```

Comment:
let's call this something like `gdasapp_chem_diagb.x`?

Reply:
Sure.